packages/testing/ts/__tests__/unit/setVerifyingKeys.test.ts

```diff
@@ -0,0 +1,70 @@
+import { deployVerifyingKeysRegistryContract, EMode, getDefaultSigner, setVerifyingKeys } from "@maci-protocol/sdk";
+import { expect } from "chai";
+import { Signer } from "ethers";
+
+import { verifyingKeysArgs } from "../../constants";
+
+describe("setVerifyingKeys", function test() {
+  this.timeout(900000);
+
+  let signer: Signer;
+  let verifyingKeysRegistryAddress: string;
+
+  before(async () => {
+    signer = await getDefaultSigner();
+  });
+
+  beforeEach(async () => {
+    verifyingKeysRegistryAddress = await deployVerifyingKeysRegistryContract({ signer });
+  });
+
+  it("should set the verifying keys of mode QV on the contract", async () => {
+    await setVerifyingKeys({ ...(await verifyingKeysArgs(signer)), verifyingKeysRegistryAddress });
+  });
+
+  it("should set the verifying keys of mode QV and NON_QV on the contract", async () => {
+    await setVerifyingKeys({
+      ...(await verifyingKeysArgs(signer, [EMode.QV, EMode.NON_QV])),
+      verifyingKeysRegistryAddress,
+    });
+  });
+
+  it("should set the verifying keys of mode QV, NON_QV and FULL on the contract", async () => {
+    await setVerifyingKeys({
+      ...(await verifyingKeysArgs(signer, [EMode.QV, EMode.NON_QV, EMode.FULL])),
+      verifyingKeysRegistryAddress,
+    });
+  });
+
+  it("should throw when setting verifying keys twice", async () => {
+    await setVerifyingKeys({
+      ...(await verifyingKeysArgs(signer)),
+      verifyingKeysRegistryAddress,
+    });
+
+    expect(
+      setVerifyingKeys({
+        ...(await verifyingKeysArgs(signer, [EMode.NON_QV])),
+        verifyingKeysRegistryAddress,
+      }),
+    ).eventually.rejectedWith("This poll joining verifying key is already set in the contract");
```

Comment:
Why there is no await?

Reply:
When I try to put `await setVerifyingKeys( ... )` I get this lint error:

<img width="689" height="273" alt="image" src="https://github.com/user-attachments/assets/6db676b2-b4b9-4ba4-a42f-186a3b2666b1" />